File: sdkwork-app-sdk-typescript/src/types/document-batch-operation-request.ts
/** Batch operation request for document content */
export interface DocumentBatchOperationRequest {
  /** Operation type: INSERT_TEXT, DELETE_RANGE, REPLACE_ALL_TEXT, REPLACE_RANGE */
  type: string;
  /** Insert index for INSERT_TEXT */
  index?: number;
  /** Range start index for DELETE_RANGE/REPLACE_RANGE */
  startIndex?: number;
  /** Range end index for DELETE_RANGE/REPLACE_RANGE */
  endIndex?: number;
  /** Text for INSERT_TEXT/REPLACE_RANGE */
  text?: string;
  /** Search text for REPLACE_ALL_TEXT */
  searchText?: string;
  /** Replace text for REPLACE_ALL_TEXT */
  replaceText?: string;
}

File: sdkwork-app-sdk-csharp/Models/DocumentBatchOperationRequest.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class DocumentBatchOperationRequest
    {
        public string? Type { get; set; }
        public int? Index { get; set; }
        public int? StartIndex { get; set; }
        public int? EndIndex { get; set; }
        public string? Text { get; set; }
        public string? SearchText { get; set; }
        public string? ReplaceText { get; set; }
    }
}
